packages/mdc-divider/_mixins.scss

```diff
@@ -0,0 +1,7 @@
+@mixin mdc-divider-core-styles {
+  @include mdc-divider-line-color(rgba(0, 0, 0, .117));
+}
+
+@mixin mdc-divider-line-color($color) {
```

Comment:
Since the divider style property is specific to context, I think it would make sense to expose divider color in `mdc-theme/_variables.scss`. We don't need separate package for this, IMO.

Reply:
Re-opening this since it is still unresolved. I do not see a value in having a separate component for divider, it can reside in mdc-theme instead.

Reply:
Should I move the class into mdc-theme or just the rgba code?

Reply:
> Since the divider style property is specific to context, I think it would make sense to expose divider color in `mdc-theme/_variables.scss`. We don't need separate package for this, IMO.

I find myself always looking for this variable when i need to implement some kind of divider.
Sometimes end up using `$mdc-list-divider-color-on-light-bg` but it seems it should be on theme base. So it could be used everywhere

Reply:
@Richienb Yes, that is correct. This should be stored in mdc-theme package as mentioned in my previous comment.

Please create a new PR to make this change. Thanks!